### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by qwuae1**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by yonzhan** [comment]
Please fix CI issues

File: src/vm-repair/HISTORY.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
 
 Release History
 ===============
+1.0.2
+++++++
+Fix repo branch to main
```
Comment (zhoxing-ms) [inline src/vm-repair/HISTORY.rst:6]: Could you explain in detail what this description means?

File: src/vm-repair/azext_vm_repair/_validators.py [hunk 2]
```diff
@@ -14,21 +15,32 @@
...
     _get_repair_resource_tag,
+    _get_current_vmrepair_version,
     _fetch_encryption_settings,
     _resolve_api_version,
     check_extension_version,
     _check_existing_rg
 )
+# For test releases and testing
+TEST_KEY = 'a6bdff92-33b5-426f-9123-33875d0ae98b'
+PROD_KEY = '3e7130f2-759b-41d4-afb8-f1405d1d7ed9'
```
Comment (zhoxing-ms) [inline src/vm-repair/azext_vm_repair/_validators.py:33]: I would like to confirm with you whether there is a security risk in exposing these keys to the public in the open source code?

**Comment by zhoxing-ms** [comment]
Please fix the azdev linter issue: https://dev.azure.com/azclitools/public/_build/results?buildId=144943&view=logs&j=e2e497a5-ba27-5eb9-f39e-d929d56974a6&t=61cc2d02-f6f4-5007-71e9-5651933c3fe7

**Comment by zhoxing-ms** [comment]
Please take a look at this comment: https://github.com/Azure/azure-cli-extensions/pull/7436#issuecomment-2027395207

> If you want to release a new extension version, please update the version in setup.py as well

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**